spark/components/promo/vanilla/promo.stories.js

```diff
@@ -1,3 +1,143 @@
 export default {
   title: 'Components|Promo',
 };
+
+export const base = () => (`
+  <div
+  class="sprk-c-Promo  sprk-c-Promo--bordered sprk-o-Stack sprk-o-Stack--split@s"
+  data-id="promo-1"
+  >
+
+    <div 
+      class="sprk-c-Promo__content sprk-o-Stack__item sprk-o-Stack sprk-o-Stack--large"
+    >
+
+      <h3 class="sprk-b-TypeDisplayFive sprk-o-Stack__item sprk-c-Promo__title">
+        Title
+      </h3>
+
+      <div class="sprk-c-Promo__subtitle sprk-o-Stack__item">
+        Subtitle
+      </div>
+
+      <div class="sprk-b-TypeBodyTwo sprk-o-Stack__item">
+        Lorem ipsum dolor. Sit amet pede. Tempus donec et. Suspendisse
+        id inventore integer eum non enim diam habitant. Maecenas nunc
+        per lacus neque egestas. Diam quod curabitur.
+      </div>
+
+      <div class="sprk-o-Stack__item">
+        <a href="#" class="sprk-c-Button">
+          Learn More
+        </a>
+      </div>
+    </div>
+
+  </div>
+`);
+
+export const flag = () => (`
+  <div
+    class="sprk-c-Promo sprk-c-Promo--flag  sprk-c-Promo--bordered sprk-o-Stack sprk-o-Stack--split@s"
+    data-id="promo-2"
+  >
+
+    <a href="#" class="sprk-o-Stack__item sprk-o-Stack__item--fourth@s">
+      <img
+        class="sprk-c-Promo__image--flag"
+        alt="Spark placeholder image."
+        src="https://sparkdesignsystem.com/assets/toolkit/images/spark-placeholder.jpg"
```

Comment:
Sync with @RCopeland  on where to put this image in NDS. Since this img src will be invalid once we go live.